pkg/crc/input/input.go

```diff
@@ -12,7 +12,7 @@ func PromptUserForYesOrNo(message string, force bool) bool {
 	}
 	var response string
 	output.OutF(message + "? [y/N]: ")
-	fmt.Scanf("%s", response)
+	fmt.Scanf("%s", &response)
```

Comment:
As per `Scanf` function definition it take `func Scanf(format string, a ...interface{})` any value then why you want to pass the address of string instead directly this string, does't it evaluate? looks like type should be the pointer https://play.golang.org/p/VSc_P1QC77G

Reply:
Right, i found out about it today..